news/_posts/2020-03-20-ps17-patch-release-lifecycle.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "PrestaShop 1.7 patch release lifecycle"
+subtitle: "The journey of a patch release of PS 1.7, step by step"
+date:   2020-03-20 16:00:00
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - releases
+ - 1.7
+ - beta
+---
+
+
+Following previous post, here is more explanations about software lifecycle at PrestaShop, focusing on patch releases and the release process.
+
+This blog post explains:
+- The detailed lifecycle for PrestaShop patch releases
+- The decision process for patch releases
+- The Maintainance scope
+
+## Patch release lifecycle
+
+Patch releases are "maintainance" releases: they provide bug fixes and security patches, but do not provide enhancements or new features. They are part of a necessary maintenance process.
+
+### Scope of maintainance
+
+When a minor version is released, such as PrestaShop 1.7.7.0, the related branch becomes the latest and maintained branch.
+This means that, when PrestaShop 1.7.7.0 is out :
+- There will be no more PrestaShop 1.7.6 patch releases, for either bug or security issues[1]
+- There might be, if necessary, patch releases for PrestaShop 1.7.7. (which means: 1.7.7.1, 1.7.7.2 and so on ...) until next minor release is delivered (PrestaShop 1.7.8)
+
+At the moment PrestaShop 1.7.7.0 is out, PrestaShop 1.7.6 reaches its [End Of Life](https://en.wikipedia.org/wiki/End-of-life_(product)), just like all previous minor versions.
+
+
+### When do we decide to release a patch ?
+
+A patch release is scheduled when we are being reported a "trigger bug":
+- A major bug in maintained branch
+- A security issue in maintained branch
+
+As of today, since PrestaShop 1.7.7.0 is still not released, the maintained branch is `1.7.6.x`. Latest patch release is PrestaShop 1.7.6.4.
+
+This means that **we will start working on a 1.7.6.5 release if a community contributor or the QA team reports a major regression in PrestaShop 1.7.6.4 or a security issue[2].**
+
+If minor or trivial regressions are reported for PrestaShop 1.7.6, they are scheduled to be fixed in next minor version. We consider that a minor or trivial bug is not important enough to trigger a patch release process which is, as explained below, a costly process for both the PrestaShop company and the PrestaShop community.
+
+From the moment a "trigger bug" is reported, we start a 6 weeks long timer. We consider that a patch release must be delivered within these 6 weeks.
+
+### What happens in six weeks
+
+From the moment the "trigger bug" is reported, we register into the dedicated Kanban (today it means the [1.7.6. Kanban board](https://github.com/PrestaShop/PrestaShop/projects/4)) the bugs to be fixed in the patch release, whether they are trivial, minor or major.
```

Comment:
"From the moment the "trigger bug" is reported, " is a duplicate from previous paragraph, is it possible to write it another way?